Disable the -fuse-ld=gold flag. This option is a Debian/Ubuntu specific
extension to the g++ and causes upstream GCC to throw an error.

Upstream-Status: Pending

Signed-off-by: Ken Werner <ken.werner@linaro.org>

Index: qt-everywhere-opensource-src-4.8.0/src/3rdparty/webkit/Source/common.pri
===================================================================
--- qt-everywhere-opensource-src-4.8.0.orig/src/3rdparty/webkit/Source/common.pri
+++ qt-everywhere-opensource-src-4.8.0/src/3rdparty/webkit/Source/common.pri
@@ -3,12 +3,12 @@
 contains(JAVASCRIPTCORE_JIT,yes): DEFINES+=ENABLE_JIT=1
 contains(JAVASCRIPTCORE_JIT,no): DEFINES+=ENABLE_JIT=0
 
-linux-g++ {
-isEmpty($$(SBOX_DPKG_INST_ARCH)):exists(/usr/bin/ld.gold) {
-    message(Using gold linker)
-    QMAKE_LFLAGS+=-fuse-ld=gold
-}
-}
+#linux-g++ {
+#isEmpty($$(SBOX_DPKG_INST_ARCH)):exists(/usr/bin/ld.gold) {
+#    message(Using gold linker)
+#    QMAKE_LFLAGS+=-fuse-ld=gold
+#}
+#}
 
 # We use this flag on production branches
 # See https://bugs.webkit.org/show_bug.cgi?id=60824
